pkg/api/v1/conversion_generated.go

```diff
@@ -4748,6 +4758,69 @@ func convert_v1_TCPSocketAction_To_api_TCPSocketAction(in *TCPSocketAction, out
 	return nil
 }
 
+func convert_v1_ThirdPartyResource_To_experimental_ThirdPartyResource(in *ThirdPartyResource, out *experimental.ThirdPartyResource, s conversion.Scope) error {
```

Comment:
Yeah, it should definitely not be making functions like this, even with the types in v1 accidentally. Why is it making functions to convert from "api/v1" to "experimental/"?